supertokens_python/recipe/emailverification/api/implementation.py

```diff
@@ -69,9 +70,12 @@ async def generate_email_verify_token_post(self, api_options: APIOptions, user_c
         email_verify_link = (await api_options.config.get_email_verification_url(
             user, user_context)) + '?token=' + token_result.token + '&rid' + api_options.recipe_id
 
-        try:
-            await api_options.config.create_and_send_custom_email(user, email_verify_link, user_context)
-        except Exception:
-            pass
+        email_verification_email_delivery_input = TypeEmailVerificationEmailDeliveryInput(
+            user=user,
+            email_verify_link=email_verify_link,
+            user_context=user_context
+        )
+        assert api_options.email_delivery is not None
```

Comment:
You don't need this assert anymore as it will always be true

Reply:
Done.